### PR TITLE
[CHORE] Update Jitsi library version

### PR DESCRIPTION
**Pull request by Cool-fire**
@RocketChat/android

#### Changes:
 Implemented the new updated `JitsiMeetViewListener` methods, removed old methods. Implemented `view.join` instead of the old `view.LoadURLObject`

### CODE REVIEW

File: app/src/main/java/chat/rocket/android/videoconference/ui/VideoConferenceActivity.kt [hunk 1]
```diff
@@ -52,34 +53,25 @@ class VideoConferenceActivity : JitsiMeetActivity(), JitsiVideoConferenceView,
...
         logJitsiMeetViewState("Joined video conferencing", map)
 
-    override fun onConferenceWillLeave(map: MutableMap<String, Any>?) =
-        logJitsiMeetViewState("Leaving video conferencing", map)
-
-    override fun onConferenceLeft(map: MutableMap<String, Any>?) {
-        logJitsiMeetViewState("Left video conferencing", map)
-        finishJitsiVideoConference()
+    override fun onConferenceTerminated(map: MutableMap<String, Any>?) {
+        if(map!!.containsKey("error")) {
```
Comment (philipbrito) [inline app/src/main/java/chat/rocket/android/videoconference/ui/VideoConferenceActivity.kt]: Avoid using `!!`

**Review by philipbrito — CHANGES_REQUESTED** [review]

File: app/src/main/java/chat/rocket/android/videoconference/ui/VideoConferenceActivity.kt (hunk 1 above)
Comment (philipbrito) [inline app/src/main/java/chat/rocket/android/videoconference/ui/VideoConferenceActivity.kt]: Is there a place where I can find all the keys like the `error` for the map?

**Review by philipbrito — CHANGES_REQUESTED** [review]

File: app/src/main/java/chat/rocket/android/videoconference/ui/VideoConferenceActivity.kt (hunk 1 above)
Comment (Cool-fire) [reply app/src/main/java/chat/rocket/android/videoconference/ui/VideoConferenceActivity.kt]: @filipedelimabrito Actually the Jitsi documentation specified the keys for Map. if the `onConferenceTerminated` ended successfully no `error` key will be present else `error` key will be there. So based on the condition the keys will be present in the map, so here I am just checking for error key.

[https://github.com/jitsi/jitsi-meet/tree/master/android#jitsimeetviewlistener](url)

Comment (Cool-fire) [reply app/src/main/java/chat/rocket/android/videoconference/ui/VideoConferenceActivity.kt]: okay @filipedelimabrito

Comment (philipbrito) [reply app/src/main/java/chat/rocket/android/videoconference/ui/VideoConferenceActivity.kt]: Thanks for pointing it out @Cool-fire.

File: app/src/main/java/chat/rocket/android/videoconference/ui/VideoConferenceActivity.kt [hunk 2]
```diff
@@ -52,34 +53,27 @@ class VideoConferenceActivity : JitsiMeetActivity(), JitsiVideoConferenceView,
...
-                    "startWithVideoMuted" to true
-                ),
-                "context" to bundleOf(
-                    "user" to bundleOf("name" to name),
-                    "iss" to "rocketchat-android"
-                ),
-                "url" to url
-            )
-        )
+        var options = JitsiMeetConferenceOptions.Builder()
```
Comment (philipbrito) [inline app/src/main/java/chat/rocket/android/videoconference/ui/VideoConferenceActivity.kt]: Should be `val` not `var`.

File: app/src/main/java/chat/rocket/android/videoconference/ui/VideoConferenceActivity.kt [hunk 3]
```diff
@@ -52,34 +53,27 @@ class VideoConferenceActivity : JitsiMeetActivity(), JitsiVideoConferenceView,
...
-    override fun onConferenceWillLeave(map: MutableMap<String, Any>?) =
-        logJitsiMeetViewState("Leaving video conferencing", map)
-
-    override fun onConferenceLeft(map: MutableMap<String, Any>?) {
-        logJitsiMeetViewState("Left video conferencing", map)
-        finishJitsiVideoConference()
+    override fun onConferenceTerminated(map: MutableMap<String, Any>?) {
+        if(!map.isNullOrEmpty()){
+            if(map.containsKey("error")) {
+                logJitsiMeetViewState("Terminated video conferencing with error", map)
```
Comment (philipbrito) [inline app/src/main/java/chat/rocket/android/videoconference/ui/VideoConferenceActivity.kt:56]: This way you'll never finish the video conference if there is an error here.

**Review by philipbrito — CHANGES_REQUESTED** [review]

**Review by philipbrito — APPROVED** [review]
I've made the needed changes here.
Thanks for sending this PR @Cool-fire

**Pull request merged**